composer/src/test/kotlin/com/gojuno/composer/LogLineParserSpec.kt

```diff
@@ -0,0 +1,42 @@
+package com.gojuno.composer
+
+import org.assertj.core.api.Assertions.assertThat
+import org.jetbrains.spek.api.Spek
+import org.jetbrains.spek.api.dsl.context
+import org.jetbrains.spek.api.dsl.it
+
+class LogLineParserSpec : Spek({
+
+    val parser = LogLineParser()
+
+    context("parse test class and name") {
+
+        it("parses old TestRunner start logs") {
```

Comment:
Maybe give better name? "old" doesn't really explain what is old and what is new

Also, you can extract subject under test which is a log line that you parse into a separate variable for better test structure